using_images/other_images/jenkins.adoc

```diff
@@ -52,11 +52,113 @@ You can find https://github.com/openshift/origin/tree/master/examples/image-stre
+
+For the `view` `*Role*`:
+
+* hudson.model.Hudson.Read
+* hudson.model.Item.Read
+
+For the `edit` `*Role*`, in addition to the permissions available to `*view*`:
+
+* hudson.model.Item.Build
+* hudson.model.Item.Configure
+* hudson.model.Item.Create
+* hudson.model.Item.Delete
+* hudson.model.Item.Workspace
+* hudson.scm.SCM.Tag
+
+Users authenticated against {product-title} OAuth will be added to the Jenkins authorization matrix upon their first successful login.
+
+[NOTE]
+====
+The user name used to create the project Jenkins is running in will have the `admin` `*Role*` assigned to it.  So consider that the default
+administrative user for Jenkins when using {product-title} OAuth to manage authentication.
+
+The `admin` user that is pre-populated in the {product-title} Jenkins image with administrative privileges will not be honored with those
+privileges when {product-title} OAuth is used unless the {product-title} cluster administrator explicitly defines that user in the {product-title}
+identity provider and assigns the `admin` `*Role*` to the `admin` user.
+====
+
+Permissions for users in Jenkins can be changed in {product-title} after those users are initially established in Jenkins.  The OpenShift Login plugin polls the {product-title} API server for permissions and will update the permissions stored in
+Jenkins for each Jenkins user with the permissions retrieved from {product-title}.  If the Jenkins UI is used to update permissions for a Jenkins user, the permission changes will be overwritten the next time the plugin polls {product-title}.
+
+You can control how often the polling occurs with the `OPENSHIFT_PERMISSIONS_POLL_INTERVAL` environment variable.  The default polling interval when no environment variable is set is 5 minutes.
+
+The following commands creates a new Jenkins xref:../../architecture/core_concepts/pods_and_services.adoc#pods[pod] using {product-title} OAuth authentication.
+The first set uses persistent volumes:
+----
+$ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/image-streams/image-streams-centos7.json -n openshift
+$ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/jenkins-persistent-template.json -n openshift
+$ oc new-app jenkins-persistent
+----
+
+
+The second set uses an EmptyDir type volume (configuration will not persist across pod restarts):
+----
+$ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/image-streams/image-streams-centos7.json -n openshift
+$ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/jenkins-persistent-template.json -n openshift
+$ oc new-app jenkins-ephemeral
+----
+
+[NOTE]
+====
+If you instantiate the template against releases prior to 3.4 of {product-title}, standard Jenkins authentication will be used, where the template
+no longer auto-generates the password for `admin`.
```

Comment:
s/3.4/v3.4/

"standard Jenkins authentication will be used.  The default 'admin' account will exist with password 'password'"